src/modules/vtol_att_control/vtol_att_control_main.cpp

```diff
@@ -543,40 +509,16 @@ VtolAttitudeControl::parameters_update()
 	/* vtol fw permanent stabilization */
 	param_get(_params_handles.vtol_fw_permanent_stab, &_params.vtol_fw_permanent_stab);
 
-	/* vtol mc mode min airspeed */
-	param_get(_params_handles.mc_airspeed_min, &v);
-	_params.mc_airspeed_min = v;
-
-	/* vtol mc mode max airspeed */
-	param_get(_params_handles.mc_airspeed_max, &v);
-	_params.mc_airspeed_max = v;
-
-	/* vtol mc mode trim airspeed */
-	param_get(_params_handles.mc_airspeed_trim, &v);
-	_params.mc_airspeed_trim = v;
-
 	/* vtol pitch trim for fw mode */
 	param_get(_params_handles.fw_pitch_trim, &v);
 	_params.fw_pitch_trim = v;
 
-	/* vtol maximum power engine can produce */
-	param_get(_params_handles.power_max, &v);
-	_params.power_max = v;
-
-	/* vtol propeller efficiency factor */
-	param_get(_params_handles.prop_eff, &v);
-	_params.prop_eff = v;
-
-	/* vtol total airspeed estimate low pass gain */
-	param_get(_params_handles.arsp_lp_gain, &v);
-	_params.arsp_lp_gain = v;
-
 	param_get(_params_handles.vtol_type, &l);
 	_params.vtol_type = l;
 
 	/* vtol lock elevons in multicopter */
 	param_get(_params_handles.elevons_mc_lock, &l);
-	_params.elevons_mc_lock = l;
+	_params.elevons_mc_lock = (l == 1);
```

Comment:
doesnt 'true' save a clock tick?

Reply:
Yes, but this is only called at init and then once per param change.
I suppose it depends on the param, but I'd often rather treat 1 as "option enabled" and 0 (or garbage) as "option disabled".